src/renderers/webgl-fallback/nodes/GLSLNodeBuilder.js

```diff
@@ -773,6 +773,10 @@ ${vars}
 
 		return `#version 300 es
 
+${ this.getSignature() }
+
+// ${ this.material.name }
```

Comment:
Hmm.. I think this makes it difficult to share the `GPU Program` because it uses the entire code as a comparison for compilation.

Reply:
Oh well...

Reply:
Could you modify this PR just to add `this.getSignature()`?

Reply:
Done!